testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java

```diff
@@ -355,18 +355,6 @@ protected VMRunnerConfiguration createVMRunner(
   }
 
   @Override
-  public String getVMArguments(ILaunchConfiguration configuration)
-      throws CoreException {
-    StringBuilder vmArgs = new StringBuilder();
-    vmArgs.append(super.getVMArguments(configuration));
-
-    vmArgs.append(" ");
-    vmArgs.append(ConfigurationHelper.getJvmArgs(configuration));

```

Comment:
Nothing came down when I pulled again, but I deployed a new one anyway. I
confirm your change and its merge are in my log:

commit 08d55a9f734272babc1ee89f6ca7fbc641570fac
Merge: 0d4fccf 12a87f6
Author: Cedric Beust <cedric@beust.com>
Date:   Wed Dec 23 22:00:14 2015 +0400

    Merge pull request #212 from missedone/issue/211

    fixed issue #211 jvm args are appended two times

commit 12a87f64dd9a9a2b5a27c84d959ac75eec1aa85b
Author: Nick Tan <missedone@gmail.com>
Date:   Wed Dec 23 06:30:18 2015 -0800

    fixed issue #211 jvm args are appended two times

Is the new one ok?
​